app/models/query/business_entity_management.rb

```diff
@@ -0,0 +1,13 @@
+module Query
+  class BusinessEntityManagement
+    def initialize(office)
+      @office = office
+    end
+
+    def jurisdictions
+      Jurisdiction.all.map do |jurisdiction|
+        Views::OfficeBusinessEntityState.new(@office, jurisdiction)
```

Comment:
Yep, you're right, there's no need for this query object. Originally the sql query was much more complicated, wasn't it. But now it's lean, so you can just make the controller construct this array.